site/docs/security/sasl.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Authentication using SASL
+prev: ../tls
+next: ../zookeeper
+---
+
+Bookies support client authentication via SASL. Currently we only support GSSAPI (Kerberos). We will start
+with a general description of how to configure `SASL` for bookies, clients and autorecovery daemons, followed
+by mechanism-specific details and wrap up with some operational details.
+
+## SASL configuration for Bookies
+
+1. Select the mechanisms to enable in the bookies. `GSSAPI` is the only mechanism currently supported by BookKeeper.
```

Comment:
We are supporting GSSAPI (kerberos) for production.
There is support for username/password usually good for tests and staging environments.
Actually this is known as DIGEST-MD5, so passwords are not sent on the wire but only digests.
As @revans2 states we should state clearly that DIGEST-MD5 is not suitable for production.

see MD5DigestBookKeeperTest